<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>300</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Klient</string>
    </property>
    <widget class="QWidget" name="centralwidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
          <number>6</number>
        </property>
        <property name="leftMargin">
          <number>10</number>
        </property>
        <property name="topMargin">
          <number>10</number>
        </property>
        <property name="rightMargin">
          <number>10</number>
        </property>
        <property name="bottomMargin">
          <number>10</number>
        </property>
        <item>
          <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
              <number>6</number>
            </property>
            <property name="leftMargin">
              <number>0</number>
            </property>
            <property name="topMargin">
              <number>0</number>
            </property>
            <property name="rightMargin">
              <number>0</number>
            </property>
            <property name="bottomMargin">
              <number>0</number>
            </property>
            <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
                <property name="spacing">
                  <number>6</number>
                </property>
                <property name="leftMargin">
                  <number>0</number>
                </property>
                <property name="topMargin">
                  <number>0</number>
                </property>
                <property name="rightMargin">
                  <number>0</number>
                </property>
                <property name="bottomMargin">
                  <number>0</number>
                </property>
                <item>
                  <widget class="QLabel" name="recipientLabel">
                    <property name="text">
                      <string>Odbiorca:</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="recipientLineEdit"/>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QRadioButton" name="radioButton1">
                <property name="text">
                  <string>Opcja 1</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="radioButton2">
                <property name="text">
                  <string>Opcja 2</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="radioButton3">
                <property name="text">
                  <string>Opcja 3</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QTextEdit" name="notesTextEdit"/>
            </item>
            <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                  <number>6</number>
                </property>
                <property name="leftMargin">
                  <number>0</number>
                </property>
                <property name="topMargin">
                  <number>0</number>
                </property>
                <property name="rightMargin">
                  <number>0</number>
                </property>
                <property name="bottomMargin">
                  <number>0</number>
                </property>
                <item>
                  <widget class="QPushButton" name="sendButton">
                    <property name="text">
                      <string>Wyślij</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QPushButton" name="discoverButton">
                    <property name="text">
                      <string>Szukaj serwerów</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item>
              <widget class="QComboBox" name="serverComboBox"/>
            </item>
            <item>
              <widget class="QLabel" name="serverLabel">
                <property name="text">
                  <string>Dostępne serwery:</string>
                </property>
              </widget>
            </item>
          </layout>
        </item>
      </layout>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>
